code-name-taran/src/com/spaghettisoft/component/checkers/Board.java

```diff
@@ -0,0 +1,230 @@
+package com.spaghettisoft.component.checkers;
+
+public final class Board {
+
+	public final String oPawn = "o";
+	public final String xPawn = "x";
+
+	private static final int ROW_COUNT = 8;
+	private static final int COLUMN_COUNT = 8;
+
+	private boolean finished = false;
+
+	public String[][] board;
+
+	public Board() {
+		initialize();
+	}
+
+	public void draw() {
+
+		int a = 0;
+		int b = 0;
+		System.out.printf("  0 1 2 3 4 5 6 7\n");
+		for (int i = 0; i < board.length; i++) {
+
+			System.out.print(a++);
+
+			for (int j = 0; j < board[i].length; j++) {
+				String sign = " ";
+
+				if (board[i][j] != null) {
+					sign = board[i][j];
+				}
+
+				System.out.print("|" + sign);
+			}
+
+			System.out.println("|" + b++);
+		}
+		System.out.printf("  0 1 2 3 4 5 6 7\n");
+	}
+
+	public void move(Player player, Move move) {
+		// Move current player's pawn
+		board[move.getTo().y][move.getTo().x] = board[move.getFrom().y][move.getFrom().x];
+		board[move.getFrom().y][move.getFrom().x] = null;
+
+		// Capture opponent player's pawn
+		Point middlePoint = new Point((move.getTo().y - move.getFrom().y) / 2 + move.getFrom().y,
+				(move.getTo().x - move.getFrom().x) / 2 + move.getFrom().x);
+		board[middlePoint.y][middlePoint.x] = null;
+
+		if ((player.getDirection() == Direction.UP && move.getTo().y == 0)
+				|| (player.getDirection() == Direction.DOWN && move.getTo().y == 7)) {
+			board[move.getTo().y][move.getTo().x] = board[move.getTo().y][move.getTo().x].toUpperCase();
+		}
+
+		if (!hasDifferentPawns()) {

```

Comment:
@VRaptor 
This method does two things:
1. Moves a pawn
2. Prints endgame message and sets a flag to false. 

You should avoid methods that do two things, unless the method job is to orchestrate other methods. The name "move" can confuse developers trying to understand the code : )